xapian-core/configure.ac

```diff
@@ -1059,7 +1075,7 @@ dnl We need to actually check for a declaration as OS X has a dummy
 dnl implementation in the library which is not prototyped in any header.
 AC_CHECK_DECL([fdatasync(int)], [
   SAVE_LIBS=$LIBS
-  AC_SEARCH_LIBS([fdatasync], [rt], [XAPIAN_LIBS="$LIBS $XAPIAN_LIBS"])
+  AC_SEARCH_LIBS([fdatasync], [rt], [XAPIAN_LIBS="$LIBS $XAPIAN_LIBS"; XAPIAN_CMAKE_LIBS=";rt;$XAPIAN_CMAKE_LIBS"])
```

Comment:
Looks like there's an extra `;` here (which I guess may be harmless, but illustrates my concern about people getting the cmake part wrong in future changes).